### PR TITLE
Feat/verifiable claims

### PR DESCRIPTION
**Pull request by MdechampG**
Add page Verifiable Claims principles in the academy. 

Still to do:
- Add illustrations
- Add links towards pages that don't exist yet 

@ccamel can you please especially review the last part that I changed as we discussed ? 

## Summary by CodeRabbit

- **Documentation**
	- Updated the description of governance and consent rules within the OKP4 protocol to better explain the dynamic nature of blockchain rules.
	- Introduced and detailed the concept of Verifiable Credentials in decentralized identity systems, highlighting their application in the OKP4 protocol and their significance in self-sovereign governance.

### CODE REVIEW

File: docs/academy/verifiable-claims.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+---
+sidebar_class_name: hidden
+---
+
+# Verifiable Claims Principles
+
+<i>Reading time: {readingTime} min</i>
```
[the remark this answers is not in the document]
Comment (MdechampG) [reply docs/academy/verifiable-claims.md:7]: It is a variable dynamically calculated.

File: docs/academy/verifiable-claims.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+In the [W3C standard](https://www.w3.org/TR/vc-data-model/), a credential is a set of one or more **claims** made by the same entity. Credentials include an identifier and metadata describing the credential's properties, such as the issuer, the expiry date and time, a representative image, a public key to use for verification purposes, the revocation mechanism, and so on. The issuer might sign the metadata. A verifiable credential is a set of tamper-evident claims and metadata that cryptographically prove who issued it.
+
+<centre>
+![verifiable-claims-1](/img/content/academy/verifiable-claims-1.webp)
+</centre>
```
[the remark this answers is not in the document]
Comment (MdechampG) [reply docs/academy/verifiable-claims.md:23]: It doesn't work with Center instead of Centre.

[the remark this answers is not in the document]
Comment (ccamel) [reply docs/academy/verifiable-claims.md:23]: @MdechampG Mixing HTML and Markdown syntax does not work. If you want to use the `<center/>` HTML tag, you must rewrite your code from Markdown to HTML. Below is a functional example extracted from the `technical-documentation` section (note that you can also manage the maximum width of the image if necessary):

```html
<center>
  <img src="/img/content/technical-documentation/governance-hierarchy-of-norms.webp" alt="Governance - Hierarchy of Norms" style={{ maxHeight: "340px" }}></img>
</center>
```

[the remark this answers is not in the document]
Comment (ccamel) [reply docs/academy/verifiable-claims.md:23]: @coderabbitai, to make the centering function correctly, it is essential not only to change `<centre/>` to `<center/>` as you suggested but also to convert the content of the tag, which is in Markdown, into HTML using the `` tag.

[the remark this answers is not in the document]
Comment (ccamel) [reply docs/academy/verifiable-claims.md:23]: @coderabbitai show me the fixed code. :relieved:

[the remark this answers is not in the document]
Comment (ccamel) [reply docs/academy/verifiable-claims.md:23]: At first glance, it looks perfectly fine. Thanks. :+1:

File: docs/academy/rules.mdx [hunk 3]
```diff
@@ -13,8 +13,8 @@ import * as quiz from './rules-quiz.json';
...
-To handle this, the OKP4 protocol ensures a secure and reliable way to interpret rules in a decentralized manner. The rules in each Zone and the consent attached to a resource determine if a user's action is permitted. It's essential to remember that this assessment also considers the evolving state of the blockchain at the time of the request.**
+The OKP4 protocol enforces rules for governance and consent. When users want to do something in a Zone (launch a workflow to create new indicators or train an AI model, for instance), the protocol checks if it's allowed based on the rules. This decision considers the current situation, as blockchain rules and digital resources constantly change. From a computer science perspective, the challenge is dealing with complex logic involving rules, such as hierarchy, inconsistency, and dependency.
```
[the remark this answers is not in the document]
Comment (MdechampG) [reply docs/academy/rules.mdx:16]: The examples are later in the text.

File: docs/academy/verifiable-claims.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+## Registration of Verifiable Presentation in the OKP4 Protocol
+
+Once the Issuer successfully issues the Verifiable Credentials (VCs) associated with a resource or a Zone to the Holder, the Registrant will record a Verifiable Presentation (VP) in the OKP4 blockchain. A VP consists of a set of VCs. The Registrant can be either the Holder or a third party.
+
+Recording VPs in the blockchain, rather than individual VCs, ensures the confidentiality of specific properties by only revealing a restricted set of properties on the public chain.
+
+<centre>
+![verifiable-claims-4](/img/content/academy/verifiable-claims-4.webp)
+</centre>
```
[the remark this answers is not in the document]
Comment (MdechampG) [reply docs/academy/verifiable-claims.md]: Done thank you

File: docs/academy/verifiable-claims.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+Use when traveling :
+
+- Airport Verification: When traveling, the passport holder presents his or her digital passport at the airport.
+- Verifier (Customs): Customs officers use a system that verifies the passport's digital signature and metadata to confirm its authenticity.
+
+The digital signature and metadata ensure enhanced security by preventing forgery and guaranteeing data integrity. Compliance with W3C standards facilitates fast, efficient data verification by verifiers.
+
+## Verifiable claims in the OKP4 protocol
+
+The [OKP4 ontology](https://docs.okp4.network/academy/resources#ontology-how-its-represented-semantic-description) is used to describe the Dataverse. In the Dataverse, resources are identified by their DID. Resources are described by claims referenced in the ontology as Verifiable Credentials (VCs) whose subject is the resource's DID. In this way, each resource is described by a set of associated claims.
```
Comment (ccamel) [inline docs/academy/verifiable-claims.md:54]:
```suggestion
The [OKP4 ontology](https://docs.okp4.network/academy/resources#ontology-how-its-represented-semantic-description) is used to describe the Dataverse. In the Dataverse, resources are identified by their DID. Resources are described by claims referenced in the ontology as Verifiable Presentations (VPs) whose subject is the resource's DID. In this way, each resource is described by a set of associated claims.
```

File: docs/academy/verifiable-claims.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+### Resource materiality
+
+It has been established that the concrete existence of a resource depends on the presence of a claim concerning it. More specifically, **the Resource Governance VC ensures the existence of a resource in the protocol**: a resource or zone exists in the protocol only if governance is attached to it.
+These VCs play a crucial role in regulating and managing resources. They define the parameters for controlling, managing, and interacting with the resource and thus regulate its use, sharing, and modification in the dataverse. These VCs guarantee order and compliance with established resource management standards.
+
+### Claim and revocation
+
+#### Self-sovereign governance principles
+
+Resources are considered to be under "Self-Sovereign" management according to the following principles:
```
Comment (ccamel) [inline docs/academy/verifiable-claims.md:77]:
```suggestion
Resources are considered to be under "Self-Sovereign" management according to the following two principles:

```

File: docs/academy/verifiable-claims.md [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+Free Issuance of Claims: In this framework, any entity (individual, organization, service, etc.) can issue claims on the resource concerned, with the exception of governance claims, which are issued by the identities themselves.
```
Comment (ccamel) [inline docs/academy/verifiable-claims.md:78]:
```suggestion
- *Free Issuance of Claims*: In this framework, any entity (individual, organization, service, etc.) can issue claims on the resource concerned, with the exception of governance claims, which are issued by the identities themselves.
```

File: docs/academy/verifiable-claims.md [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+Unilateral revocation of claims: This principle stipulates that claims can only be revoked by the entity that issued them. This means that if Entity A issues a claim on the resource, only Entity A can revoke that claim. This principle is unalterable and remains independent of any intervention or modification by governance structures.
```
Comment (ccamel) [inline docs/academy/verifiable-claims.md:79]:
```suggestion
- *Unilateral revocation of claims*: This principle stipulates that claims can only be revoked by the entity that issued them. This means that if Entity A issues a claim on the resource, only Entity A can revoke that claim. This principle is unalterable and remains independent of any intervention or modification by governance structures.

```

File: docs/academy/verifiable-claims.md [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+
+Verifiable Credentials obey schemas that describe a particular aspect of a resource. Certain VCs apply depending on the nature of the resource. A VC describing an entity or organization does not apply to the same resource as a VC describing a dataset. This is why VCs are organized in schemas.
+
+<center>
+  <img src="/img/content/academy/verifiable-claims-3.webp" alt="Verifiable claims attached to a resource" style={{ maxHeight: "400px" }}></img>
+</center>
+
+### Resource materiality
+
+It has been established that the concrete existence of a resource depends on the presence of a claim concerning it. More specifically, **the Resource Governance VC ensures the existence of a resource in the protocol**: a resource or zone exists in the protocol only if governance is attached to it.
```
Comment (ccamel) [inline docs/academy/verifiable-claims.md:70]:
```suggestion
It has been established that the concrete existence of a resource depends on the presence of a claim concerning it. More specifically, **the Resource Governance VC ensures the existence of a resource in the protocol**: a resource or zone exists in the protocol only if governance is attached to it.

```

File: docs/academy/verifiable-claims.md (hunk 7 above)
Comment (ccamel) [inline docs/academy/verifiable-claims.md:78]: I think it's important to emphasise that this framework only applies to VCs that are not governance VCs. It's written, but not emphasised enough, imho.

**Review by ccamel — CHANGES_REQUESTED** [review]
That's great. :relaxed:  A few other minor comments to take into account as you wish.

File: docs/academy/verifiable-claims.md (hunk 7 above)
Comment (MdechampG) [reply docs/academy/verifiable-claims.md]: Changed thanks

Comment (MdechampG) [reply docs/academy/verifiable-claims.md]: Indeed, I highlighted it.

**Review by ccamel — APPROVED** [review]
Great! :+1:

**Pull request merged**